mlir-tensorrt/tensorrt/lib/TensorRT/Transforms/BroadcastElimination.cpp

```diff
@@ -263,6 +263,11 @@ static bool broadcastAbsorbPreconditions(BroadcastOp broadcastOp) {
 }
 
 namespace {
+
```

Comment:
This change seems to be causing a regression in `broadcast_dynamic_expand_shape_regression` but I haven't looked into why yet.